boefjes/boefjes/plugins/models.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, path: Path, package: str):
         self.boefje.produces = self.boefje.produces.union(set(_default_mime_types(self.boefje)))
         self.module: Runnable | None = None
 
-        if (path / ENTRYPOINT_BOEFJES).exists():
+        if (path / ENTRYPOINT_BOEFJES).exists() and self.boefje.oci_image is None:
```

Comment:
lets check for the image first, thats a cheaper call.

Reply:
Good catch, fixed in [6530037](https://github.com/minvws/nl-kat-coordination/pull/3621/commits/65300372c14dfa7306e3114d204cf1980c1408f2)